docs/api/sentence.md

```diff
@@ -0,0 +1,14 @@
+# sentence
+
+This module contains libraries for sentence segmentation and sentence corpus construction
+
+## Module Overview
+
+::: remarx.sentence
+
+## Segmentation Functions
+
+::: remarx.sentence.segment
+options:
+show_source: true
+heading_level: 3
```

Comment:
This isn't working properly - the options display as text in the output.  

I prefer not to show source, and prefer options to be configured in one central place if we can.